web/modules/custom/brms_migrate/migrations/brms_migrate_cfrs.yml

```diff
@@ -0,0 +1,33 @@
+langcode: en
```

Comment:
I've always wondered how the "migrations" folder works, do they get picked up automatically?

Reply:
I've used migrate_manifest for this in the past

Reply:
They get picked up automatically and the beauty of them is you don't have to reimport config every time you make a change to the definition. Very useful during development. One downside is that they don't show up in the list of migrations in drupal UI so you can't run them from there. `drush migrate:status` does list them.